test/unit/traits/src/NativeHandleTest.cpp

```diff
@@ -17,5 +17,7 @@ TEMPLATE_LIST_TEST_CASE("NativeHandle", "[handle]", alpaka::test::TestAccs)
     auto handle = alpaka::getNativeHandle(dev);
 
     STATIC_REQUIRE(std::is_same_v<alpaka::NativeHandle<Dev>, decltype(handle)>);
-    STATIC_REQUIRE(std::is_same_v<alpaka::NativeHandle<Dev>, int>); // It won't work for SYCL backend
+    /* The SYCL backend does not use an int as the native handle type
+    STATIC_REQUIRE(std::is_same_v<alpaka::NativeHandle<Dev>, int>);
+    */
```

Comment:
This is different from the version in #1845. Is this intended?

Reply:
I prefer this approach, because either a requirement is there or it isn't.

But I'm fine with either one (also because I expect we will review the requirement after #1988 is merged), let me know what you prefer.

Reply:
Okay, then I'd prefer aligning with #1845. One less file to care about in that mega PR :-)

Reply:
Done.

Reply:
(and rebased)